src/main/steps/applicant1/hub-page/progress-bar.njk

```diff
@@ -43,32 +43,32 @@
     label: {
       text: response
     },
-    complete: progressionIndex > 4,
-    active: progressionIndex >= 0 and progressionIndex < 5
+    complete: progressionIndex > 7,
```

Comment:
you added 5 different states but this went up by 3, is this correct?

Reply:
@velimd yeah that was the main fix for the ticket, the response circle wasn't showing up as complete when the state was Holding because the number was 2 too high